railties/lib/rails/commands/server/server_command.rb

```diff
@@ -132,7 +170,17 @@ class ServerCommand < Base # :nodoc:
       def initialize(args = [], local_options = {}, config = {})
         @original_options = local_options
         super
-        @server = self.args.shift
+        @server = if using
+          ActiveSupport::Deprecation.warn(<<-MSG.squish)
```

Comment:
Why not use `<<~` here?

Reply:
It still needs the squish; deprecation messages should not be hard-wrapped on output.